### PR TITLE
gh-26 Add support for DPR models

### LINKED ISSUE #26: Add support for DPR models

### PR DESCRIPTION
**Pull request by antas-marcin**
Closes #26

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -60,6 +60,26 @@ jobs:
         MODEL_TAG_NAME: sentence-transformers-gtr-t5-large
       if: tag IS present
       stage: buildanddeploy
+    - env:
+        MODEL_NAME: vblagoje/dpr-ctx_encoder-single-lfqa-wiki
+        MODEL_TAG_NAME: vblagoje-dpr-ctx_encoder-single-lfqa-wiki
+      if: tag IS present
+      stage: buildanddeploy
+    - env:
+        MODEL_NAME: vblagoje/dpr-question_encoder-single-lfqa-base
```
Comment (vblagoje) [inline .travis.yml:69]: @antas-marcin here we also use wiki not base

File: .travis.yml [hunk 2]
```diff
@@ -60,6 +60,26 @@ jobs:
...
+        MODEL_TAG_NAME: vblagoje-dpr-question_encoder-single-lfqa-base
```
Comment (vblagoje) [inline .travis.yml:70]: And here as well

File: .travis.yml (hunk 1 above)
Comment (antas-marcin) [reply .travis.yml]: I have fixed that, thanks for checking this @vblagoje

**Comment by vblagoje** [comment]
LGTM @antas-marcin

**Pull request merged**